idx/cassandra/config.go

```diff
@@ -0,0 +1,115 @@
+package cassandra
+
+import (
+	"errors"
+	"flag"
+	"time"
+
+	"github.com/rakyll/globalconf"
+	log "github.com/sirupsen/logrus"
+)
+
+type IdxConfig struct {
+	Enabled          bool
+	pruneInterval    time.Duration
+	updateCassIdx    bool
+	updateInterval   time.Duration
+	updateInterval32 uint32
+
+	writeQueueSize int
+
+	ssl                      bool
+	auth                     bool
+	hostverification         bool
+	createKeyspace           bool
+	schemaFile               string
+	keyspace                 string
+	hosts                    string
+	capath                   string
+	username                 string
+	password                 string
+	consistency              string
+	timeout                  time.Duration
+	numConns                 int
+	protoVer                 int
+	disableInitialHostLookup bool
+}
+
+const timeUnits = "Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'"
```

Comment:
in a file, please put first global consts and vars, then types followed by their methods